dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraMetaManager.java

```diff
@@ -116,7 +118,10 @@ public Optional<FileStatus> getFromUfs(String path) throws IOException {
       UnderFileSystem ufs = getUfsInstance(path);
       UfsStatus status = ufs.getStatus(path,
           GetStatusOptions.defaults().setIncludeRealContentHash(mGetRealContentHash));
-      Map<String, String> xattrMap = ufs.getAttributes(path);
+      Map<String, String> xattrMap = null;
+      if (mXAttrWriteToUFSEnabled) {
+        xattrMap = ufs.getAttributes(path);
+      }
       DoraMeta.FileStatus fs = mDoraWorker.buildFileStatusFromUfsStatus(status, path, xattrMap);
```

Comment:
hmmm then add a `@Nullable` on `mDoraWorker.buildFileStatusFromUfsStatus(status, path, @Nullable xattrMap);`

Reply:
the method already has the `@Nullable` for xattrMap, what do you mean? for the method?

Reply:
ok got it